.eslintrc.cjs

```diff
@@ -18,6 +18,7 @@ module.exports = {
     'digitalbazaar/module',
   ],
   rules: {
-    'jsdoc/check-examples': 0
+    'jsdoc/check-examples': 0,
+    'max-len': ['error', {code: 80, ignoreUrls: true}]
```

Comment:
`ignoreUrls: true` looks like another good one to go upstream: https://github.com/digitalbazaar/eslint-config-digitalbazaar/pull/77